src/proto/factories/layer_graph_factory.cpp

```diff
@@ -172,6 +172,11 @@ construct_layer_graph(lbann_comm* comm,
 
 #undef TEMPLATE_INSTANTIATION
 
+    // Check that layer has been constructed
+    if (l == nullptr) {
+      LBANN_ERROR("Could not construct layer ", name);
```

Comment:
Why is that?

Reply:
If it's null, it means the instantiation attempts (generated by macros) failed, as the default value of `l` is `nullptr`.